draft-ietf-mmusic-t140-usage-data-channel.xml

```diff
@@ -38,8 +38,8 @@
         transport mechanism for Real-time text using the ITU-T Protocol for multimedia
         application text conversation (Recommendation ITU-T T.140), and
         how the SDP offer/answer mechanism can be used to negotiate
-        such data channel, referred to as T.140 data channel. The document updates
-        RFC 8373.
+        such data channel, referred to as T.140 data channel. This document updates
+        RFC 8373 to specify its use with WebRTC data channels.
```

Comment:
Change to 
RFC 8373 to specify its use with WebRTC, T.140 data channels.

(I just commented another version of this wording. We only specify the language modality for t140 subchannels.)

Reply:
We update 8373 to specify its use with WebRTC data channels. We don't say that all data channels have to use the same modality.